hammett/widgets/base.py

```diff
@@ -41,7 +41,25 @@
     from hammett.widgets.types import Choice, Choices, InitializedChoices
 
 
-class BaseStateWidget(Screen):
+class BaseWidget(Screen):
+    """The class implements the base interface for widgets from the library."""
```

Comment:
Replace it with `This class defines the base interface for all Hammett widgets.`